src/site/filters/liquid.js

```diff
@@ -390,6 +390,15 @@ module.exports = function registerFilters() {
     return fieldLink;
   };
 
+  liquid.filters.pathBCFormat = (bc, string) => {
+    const last = { url: { path: bc.path, routed: true }, text: string };
+    if (bc.length > 0) {
+      bc.push(last);
+    }
+
+    return bc;
+  };
```

Comment:
Sorry, this is a little hard for me to follow...Can we rename this filter to `deriveLastBreadcrumbFromPath`? Is that an accurate name? And instead of `bc`, can we use `breadcrumbs` as the var name? 

I'm also not clear on what `if (bc.length > 0) {` does because I don't see when a page wouldn't have any breadcrumbs?

Reply:
Definitely thought we were talking about bitcoin when I first read this 😂

Reply:
Shouldn’t we at least have unit tests for the code in this file? It’s just Javascript.

Reply:
@mpelzsherman that seems like a great idea, I don't think we have tests for liquid filters yet but I think they'd be invaluable! 👏

Reply:
Yeah, this is unnecessary, @ncksllvn :+1: